pom.xml

```diff
@@ -31,52 +32,52 @@
 		<selenium.version>2.45.0</selenium.version>
 	</properties>
 
-    <repositories>
-        <repository>
-            <id>devhub</id>
-            <url>https://devhub.ewi.tudelft.nl/nexus/content/repositories/releases/</url>
-            <releases>
-                <enabled>true</enabled>
-                <updatePolicy>always</updatePolicy>
-            </releases>
-            <snapshots>
-                <enabled>false</enabled>
-            </snapshots>
-        </repository>
-    </repositories>
+	<repositories>
```

Comment:
While consistent idententation is desirable, it's more important to find which commit introduced a single line when you want to find which commit introduced a problem. Therefore, I think, indentation should only change if that part of the code is changed.